### PR TITLE
feat: nyx integration

### PR DESCRIPTION
**Pull request by hedi-edelbloute**
### 📝 Description

Integrate nyx

### ❓ Context

- **Impacted projects**: `LLC, LLD, LLM` 
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-7319` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by henrily-ledger** [comment]
Could you also add nyx in spec.ts and if possible add it to bridge.integration and his own dataset ?

**Comment by henrily-ledger** [comment]
code LGTM but there's some CI Failure (i didn't check)

File: libs/ledger-live-common/src/families/cosmos/js-getTransactionStatus.ts [hunk 1]
```diff
@@ -160,7 +160,7 @@ export class CosmosTransactionStatusManager {
 
     const estimatedFees = t.fees || new BigNumber(0);
 
-    if (!t.fees || !t.fees.gt(0)) {
+    if (t.fees == null) {
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/cosmos/js-getTransactionStatus.ts:163]:
```suggestion
    if (t.fees === null) {
```

File: libs/ledger-live-common/src/families/cosmos/js-getTransactionStatus.ts [hunk 2]
```diff
@@ -231,7 +231,7 @@ export class CosmosTransactionStatusManager {
     }
 
     const estimatedFees = t.fees || new BigNumber(0);
-    if (!t.fees || !t.fees.gt(0)) {
+    if (t.fees == null) {
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/cosmos/js-getTransactionStatus.ts:234]:
```suggestion
    if (t.fees === null) {
```

**Comment by gre** [comment]
👋  ESLint config has been unified on develop after we merged https://github.com/LedgerHQ/ledger-live/pull/3622 – this will improve the project consistency, remove eslint dependencies duplicates, and overall simplify the maintenance. This comes with a prettier library upgrade and unification of the prettierrc too, so you likely have conflicts with develop.


Hopefully, here is the procedure to easily fix this. **please proceed as follow**:

**Step 1: sync your branch with develop up to the config changes:**

2 cases

- if your branch were continuously rebased with develop and don't have any merge commit, you can do 
```
git rebase aa1e0ebc0a0498b4350754899b6a368fbd42e811
```
- otherwise prefer 
```
git merge aa1e0ebc0a0498b4350754899b6a368fbd42e811
```

**then, apply the config**

```
pnpm i
pnpm lint:fix --force
```

if there are remaining errors that lint couldn't fix, please fix them and `pnpm lint` again.

then commit in your branch.

```
git commit -a -m "chore: lint fixes"
```

**Step 2: finalize the rebase/merge with develop**

2 cases

- if your branch were continuously rebased with develop and don't have any merge commit, you can do
```
git pull --rebase origin develop
```
- otherwise prefer 
```
git merge origin/develop
```

you should now be up to date. 👍

---

*Thanks and sorry for the inconvenience,*
*gre*

**Pull request closed without merging**